src/anyon/anyon.jl

```diff
@@ -32,6 +34,7 @@ struct AnyonYukonQPU <: AbstractQPU
     status_request_throttle::Function
     connectivity::LineConnectivity
     project_id::String
+    metadata::Vector{Metadata}
```

Comment:
Why is metadata a vector?

Reply:
it was to get aroung the fact that struct fields are immutable, but I can also mutate an empty Dict to implement the lazy fetch. Updated